src/main/scala/uk/gov/nationalarchives/consignmentexport/Main.scala

```diff
@@ -18,6 +18,7 @@ import uk.gov.nationalarchives.consignmentexport.Config.config
 import uk.gov.nationalarchives.consignmentexport.GraphQlApi.backend
 import uk.gov.nationalarchives.consignmentexport.StepFunction.ExportOutput
 import uk.gov.nationalarchives.tdr.keycloak.TdrKeycloakDeployment
+import uk.gov.nationalarchives.consignmentexport.BuildInfo.version
```

Comment:
On line 21, the `Main` object signature has a `version` parameter defaulting to 0.0.1, is this needed anymore?

Reply:
After speaking with Sam, no, it shouldn't; he said to replace it with the BuildInfo version so I'll do that.